modules/IngestDrugSynonymsSync.py

```diff
@@ -0,0 +1,228 @@
+import requests
+import pandas as pd
+import re
+import json
+import sys
+from pathlib import Path
+import xlrd
+import csv
+import os
+import tempfile
+import numpy as np
+from typing import Optional
+from functools import partial
+import logging
+logger = logging.getLogger('ds')
+
+
+class IngestDrugSynonyms():
+
+    def __init__(self, configPath: Path = Path("config.json")):
+        self.check_config(configPath)
+        self.url_international: str = os.environ["URL_INT"] if isinstance(
+            os.environ["URL_INT"], str) else None
+        self.url_USA: str = os.environ["URL_USA"] if isinstance(
+            os.environ["URL_USA"], str) else None
+        self.url_drugbank: str = os.environ["URL_DRUGBANK"] if isinstance(
+            os.environ["URL_DRUGBANK"], str) else None
+        self.query_keywords: [] = os.environ["QUERY_KEYWORDS"].split(
+            ",") if isinstance(os.environ["QUERY_KEYWORDS"], str) else None
```

Comment:
Was a little surprised to see configfile / envvar  stuff here vs. regular parameters (ex: testability), but ok!